src/Service/ReportImporter.php

```diff
@@ -117,12 +101,12 @@ public function import($src)
             }
         }
 
-        // Archive reports that no longer exist in Systemportalen.
+        // Archive reports that no longer exist in anmeldelsesportalen.
         $this->reportRepository->createQueryBuilder('e')
             ->update()
             ->set('e.archivedAt', ':now')
             ->setParameter('now', new \DateTime(), Type::DATETIME)
-            ->where('e.sysId NOT IN (:sysIds)')
+            ->where('e.sysInternalId NOT IN (:sysIds)')
```

Comment:
Is it a bug fix or an improvement that we now use `sysInternalId` in stead of `sysId`?